case-lib/hijack.sh

```diff
@@ -54,9 +54,18 @@ function func_exit_handler()
         # shellcheck disable=SC2154
         logfile="$logfile"
 
-        local loggerBin wcLog; loggerBin=$(basename "$SOFLOGGER")
-        # We need this to avoid the confusion of a "Terminated" message
+        local loggerBin wcLog logtool; loggerBin=$(basename "$SOFLOGGER");
```

Comment:
Don't declare `logtool` 30 lines before the first time it's used, it's less readable.